packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml

```diff
@@ -0,0 +1,44 @@
+<mjml
+  lang="en"
+>
+  <mj-head>
+    <mj-font name="PT Sans"
+      href="https://fonts.googleapis.com/css?family=PT+Sans" />
+
+    <mj-attributes>
+      <mj-class name="body-text" font-size="14px" color="#646464" font-family="PT Sans" align="center" font-weight="400" />
+      <mj-class name="title" font-size="20px" font-size="20px" color="#000" align="center" font-weight="900" />
+      <mj-all font-family="PT Sans"/>
+      <mj-button background-color="#0061A8" href="#" text-transform="uppercase" />
+    </mj-attributes>
+  </mj-head>
+  <mj-body background-color="white">
+    <mj-section>
+      <mj-column>
+        <mj-image width="200px" src="https://web-components.prod.paas.reapit.cloud/reapit-connect.jpeg" alt="Reapit Connect Logo" />
+      </mj-column>
+    </mj-section>
+    <mj-section>
+      <mj-column>
+        <mj-text mj-class="title">
+          Welcome to Reapit Connect
+        </mj-text>
+
+        <mj-text mj-class="body-text">Hi {userName},</mj-text>
+        <mj-text mj-class="body-text">Welcome to Reapit Connect.</mj-text>
+        <mj-text mj-class="body-text">Reapit Connect is our single sign on solution which allows you to seamlessly access products and services provided by Reapit Ltd.</mj-text>
+        <mj-text mj-class="body-text">Please see below your userName and temporary password. When you click the login link, you will be re-directed to a screen where you will be asked to change your password.</mj-text>
+        <mj-text mj-class="body-text"><strong>User Name: {userName}</strong></mj-text>
+        <mj-text mj-class="body-text"><strong>Password: {code}</strong></mj-text>
```

Comment:
Not sure on this value? I think AWS expects it to be {###} ? Not entirely sure though